pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -0,0 +1,133 @@
+import os
+import re
+import subprocess
+
+import click
+import koji
+import yaml
+from errata_tool import Erratum
+from pyartcd import constants
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+from pyartcd.runtime import Runtime
+
+
+class OperatorSDKPipeline:
+    def __init__(self, runtime: Runtime, group: str, assembly: str, updatelatest: bool) -> None:
+        self.runtime = runtime
+        self._logger = runtime.logger
+        self.version = group.split("-")[1]
+        self.assembly = assembly
+        self.updatelatest = updatelatest
+        self.sdk = "operator-sdk"
+
+        self._jira_client = runtime.new_jira_client()
+        ocp_build_data_repo = runtime.new_github_client().get_repo("openshift/ocp-build-data")
+        release_file = ocp_build_data_repo.get_contents("releases.yml", ref=group)
+        release_yaml = yaml.load(release_file.decoded_content, Loader=yaml.FullLoader)
+        self.extra_ad_id, self.parent_jira_key = self.get_ad_jira_key(assembly, release_yaml)
+
+    def run(self):
+        advisory = Erratum(errata_id=self.extra_ad_id)
+        self._logger.info("Check advisory status ...")
+        if advisory.errata_state in ["QE", "NEW_FILES"]:
+            self._logger.info("Advisory status not in REL_PREP yet ...")
+            return
+        if advisory.errata_state == "SHIPPED_LIVE":
+            self._logger.info("Advisory status already in SHIPPED_LIVE, update subtask 9 ...")
+            self._update_jira(self.parent_jira_key, 8, "Advisory status already in SHIPPED_LIVE")
+        self._logger.info("Advisory status already in post REL_PREP, update subtask 7 ...")
+        self._update_jira(self.parent_jira_key, 6, "Advisory status already in REL_PREP")
+
+        et_builds = advisory.errata_builds
+        sdk_build = [i for i in et_builds[f'OSE-{self.version}-RHEL-8']
```

Comment:
I think you could iterate all product versions here. We will have rhel 9 builds and we don't want this to break.

Reply:
Otherwise, you could use `elliott -i $something find-builds -k image` to find the NVR.

Reply:
this job is only for operator-sdk usage, this image only have rhel8 product list now, if product list changed, we will get a request

Reply:
I am not convinced to hardcode the product version `OSE-{self.version}-RHEL-8` here. It is pretty easy to find the sdk build regardless of the product version.